sessions/css-selectors/demo-end/css/styles.css

```diff
@@ -54,9 +34,9 @@ of an HTML tag. Like <section class="main-section">. */
 }
 
 .main-section {
-  padding: var(--gap);
-  background-color: var(--color-water-10);
-  margin-bottom: var(--gap);
+  padding: 20px;
+  background-color: lavenderblush;
```

Comment:
Strictly speaking, the general consensus is to use hex colors instead of color names. I don't really mind either way. But since we're using hex codes for the neuefische colors, we should probably keep using hex codes.

Reply:
Ok, then hex colors. In the previous demo it was a mix of hex colors and color names. 🙂